contracts/staking/test/actors/finalizer_actor.ts

```diff
@@ -187,8 +216,9 @@ export class FinalizerActor extends BaseActor {
     private async _getOperatorShareByPoolIdAsync(poolIds: string[]): Promise<OperatorShareByPoolId> {
         const operatorShareByPoolId: OperatorShareByPoolId = {};
         for (const poolId of poolIds) {
-            const pool = await this._stakingApiWrapper.stakingContract.getStakingPool.callAsync(poolId);
-            operatorShareByPoolId[poolId] = new BigNumber(pool.operatorShare);
+            operatorShareByPoolId[poolId] = new BigNumber(
```

Comment:
Turns out this was actually less legible with `_.zipObject` because it needs to extract the `operatorShare` from the call result, so `_.zipObject(poolIds, (await Promise.all(pooIds.map(id => getPoolAsync(id))).map(p => p.operatorShare))`